update-header/action.yaml

```diff
@@ -22,13 +22,6 @@ inputs:
   directories:
     description: "Specify the directories that should be updated, separated by space e.g. 'foo bar baz'"
     required: true
-  exclude-file:
-    description: "Specify the file, that contains files to ignore"
-    default: .pontos-header-ignore
-    required: false
```

Comment:
Why have you removed this?
I think this is/was required by some teams ...

Reply:
the `exclude-file` input is not used in the workflow. should it be added?